### ISSUE TITLE
RADIUS functions and utilities 

### ISSUE THREAD

**Issue by mtnocean** [finish06/pyunifi#52]
Hi,
I've written some code for setting RADIUS usernames and passwords along with some demo Python code. Please tell me if you would like me to submit a pull request.
Paul

https://github.com/mtnocean/pyunifi/tree/copyRADIUSsite

functions:
- ``get_radius_users(self)``
- ``add_radius_user(self, name, password)``
- ``update_radius_user(self, name, password, id)``
- ``delete_radius_user(self, id)``

support functions for delete_radius_user():
- ``_delete(self, url, params=None)``
- ``_api_delete(self, url, params=None)``

utilities:
- ``unifi-ls-radius`` - lists all usernames/passwords
- ``unifi-save-radius`` - saves all usernames/passwords to a .csv file
- ``unifi-copy-radius`` - copies all usernames/passwords from one site to another

in progress:
- ``unifi-upload-radius`` - uploads a .csv file of usernames/passwords to a site

**Comment by finish06** [comment]
@mtnocean Indeed.  Please submit a PR.  I will look it all over, and check everything out.  Have a great day.
